lib/commands/os/versions.ts

```diff
@@ -55,11 +55,21 @@ export default class OsVersionsCmd extends Command {
 	public async run() {
 		const { args: params } = this.parse<FlagsDef, ArgsDef>(OsVersionsCmd);
 
-		const { versions: vs, recommended } =
-			await getBalenaSdk().models.os.getSupportedVersions(params.type);
+		const vs =
+			(await getBalenaSdk().models.hostapp.getAllOsVersions([params.type]))[
```

Comment:
If you bump to the latest, you can instead use `getAvailableOsVersions()`, since after the next major, `getAllOsVersions()` is also going to include invalidated versions as well.

Reply:
Done 👍